base/uk.ac.stfc.isis.ibex.epics/src/uk/ac/stfc/isis/ibex/epics/observing/ClosableObservable.java

```diff
@@ -73,14 +71,10 @@ public void close() {
      */
     protected void setValue(T value) {
         currentError = null;
-        if (value == null) {
-            return;
-        }
-
-        this.value = value;
-
-        for (Observer<T> observer : observers) {
-            observer.onValue(value);
+        if (value != null) {
+        	this.value = value;
+        	for (Observer<T> observer : observers)
+        		observer.onValue(value);
```

Comment:
Delimit for-loop body with braces as per our code style

Reply:
Will do, as above